src/content/getusermedia/volume/js/main.js

```diff
@@ -36,16 +36,22 @@ function successCallback(stream) {
   // browser console.
   window.stream = stream;
   var soundMeter = window.soundMeter = new SoundMeter(window.audioContext);
-  soundMeter.connectToSource(stream);
-
-  setInterval(function() {
-    instantMeter.value = instantValueDisplay.innerText =
-        soundMeter.instant.toFixed(2);
-    slowMeter.value = slowValueDisplay.innerText =
-        soundMeter.slow.toFixed(2);
-    clipMeter.value = clipValueDisplay.innerText =
-        soundMeter.clip;
-  }, 200);
+  soundMeter.connectToSource(stream, function (e) {
```

Comment:
Please fix all the style issues as you can see in the test run: https://travis-ci.org/webrtc/samples/jobs/117060901.